debugger/src/main/kotlin/org/rust/debugger/runconfig/RsDebugRunner.kt

```diff
@@ -185,8 +191,12 @@ private fun buildProjectAndGetBinaryArtifactPath(project: Project, command: Carg
                                     || kind == TargetKind.TEST
                                     || (kind == TargetKind.LIB && profile.test)
                             }
-                            .flatMap { it.filenames.filter { !it.endsWith(".dSYM") } } // FIXME: correctly launch debug binaries for macos
-                            .let(DebugBuildResult::Binaries)
+                            .filter { (_, profile) -> !testsOnly || profile.test }
```

Comment:
Maybe merge this `filter` with previous? Looks like both conditions are about similar things